tests/data/claypit/tests/Attrs/BasicScenarioCest.php

```diff
@@ -39,7 +39,8 @@ private function close1(AttrsTester $I)
     }
 
     #[Group('e1')]
-    #[Examples([1, 1], [2, 2])]
+    #[Examples(1, 1)]
+    #[Examples(2, 2)]
```

Comment:
As already mentioned in https://github.com/Codeception/Codeception/pull/6752#discussion_r1605002615 this is a slight BC break introduced by this PR.

However, the behavior doesn't seem to be documented and from DX perspective I would prefer the new behavior.

Any thoughts about this are welcome.

Reply:
The first syntax allows for dereferencing from arrays using `...`. also what about generators? They can't use the 2nd Syntax

Reply:
> The first syntax allows for dereferencing from arrays using `...`.

Shouldn't this still be possible by just passing an array as example, f.e. `#[Examples(['foo' => 'bar'])]`? Please correct me if I misunderstood the problem.

> what about generators? They can't use the 2nd Syntax

Do you mean that the new syntax breaks generators? Can you provide an example?